handlers/main.yml

```diff
@@ -1,3 +1,7 @@
 ---
 - name: Restart marathon
   service: name=marathon state=restarted
+
+- name: Reload daemon
+  command: systemctl daemon-reload
```

Comment:
I don't want to require higher Ansible version for this change alone, let's give it some time.